### PR TITLE
Added release notes for 6.2.2 and 7.0 Preview 7

### PR DESCRIPTION
**Pull request by kkeirstead**

### CODE REVIEW

File: documentation/releaseNotes/releaseNotes.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
...
 
-- ⚠️ [Here is a breaking change we did and its work item] (#737)
-- [Here is a new feature we added and its work item] (#737)
-
-\*⚠️ **_indicates a breaking change_**
\ No newline at end of file
+- Delete endpoint on startup (#2178)
+- Fix duplicated values for `/metrics` (#2149)
+- Add `CollectLiveMetrics` collection rule action (#2119)
+- Convert logger string format and arguments to native platform string width (#2184)
```
Comment (jander-msft) [inline documentation/releaseNotes/releaseNotes.md:6]: This should be removed as it does not impact the release.

Comment (kkeirstead) [reply documentation/releaseNotes/releaseNotes.md:6]: That's what I figured, but decided it was better to put it here and then remove it than for it to be forgotten. I'll pull it out

File: documentation/releaseNotes/releaseNotes.md [hunk 2]
```diff
@@ -1,6 +1,5 @@
 Today we are releasing the next official preview of the `dotnet monitor` tool. This release includes:
 
-- ⚠️ [Here is a breaking change we did and its work item] (#737)
-- [Here is a new feature we added and its work item] (#737)
-
-\*⚠️ **_indicates a breaking change_**
\ No newline at end of file
+- Delete endpoint on startup (#2178)
+- Fix duplicated values for `/metrics` (#2149)
+- Add `CollectLiveMetrics` collection rule action (#2119)
```
Comment (jander-msft) [inline documentation/releaseNotes/releaseNotes.md:5]: Let's add in the fixes in #2149 and #2165, something like:

```
- Fixed memory leak in authentication handler (#2165)
- Fixed duplication of metrics (#2149)
```

and to the 6.2.2 release (with its corresponding PR numbers)

Comment (kkeirstead) [reply documentation/releaseNotes/releaseNotes.md:6]: I already had #2149 in there, but I switched the wording to what you put and added in the other one (I wasn't sure we were going to mention that in the release notes)

**Review by jander-msft — APPROVED** [review]

**Pull request merged**